examples/slack-kit-example/README.md

```diff
@@ -1,38 +1,314 @@
+
+On this screen, you can initiate the OAuth flow with Slack. This page uses two SlackKit components to help facilitate this interaction: `SlackAuthButton` and `SlackAuthContainer`.
+
+![initiate Slack OAuth](./images/initiate-slack-auth.png)
+
+Both of these components are client-side components. The `SlackAuthButton` component takes your `redirectUrl` and `slackClientId` as props:
+
+```
+<SlackAuthContainer
+    actionButton={
+        <SlackAuthButton
+        slackClientId={process.env.NEXT_PUBLIC_SLACK_CLIENT_ID!}
+        redirectUrl={process.env.NEXT_PUBLIC_REDIRECT_URL}
+        />
+    }
+/>
+```
+
+When this button is clicked, it will initiate an OAuth flow in a pop-up window where the user can authenticate with Slack and choose a workspace to install the app. Once the flow is complete, the pop-up window should close and the state of the component should update to show it is `Connected` to Slack. Behind the scenes, Knock handles the OAuth callback from Slack and stores an `access_token` on the `tenant` you provided to the `KnockSlackProvider` as [channel data](https://docs.knock.app/managing-recipients/setting-channel-data).
+
+**Note: if the pop-up window does not close, double check that the `redirectUrl` matches your current environment.**
+
+From here, you can click `Next` to choose one or more Slack channels to associate with your object recipient.
+
+## Choosing Slack channels
+
+On this route, you can associate your Knock object recipient with one or more Slack channels using the `SlackChannelCombobox` component. It accepts a `slackChannelsRecipientObject` as a prop that specifies a `collection` and `objectId` for the recipient object:
+
+```
+"use client";
+
+import { SlackChannelCombobox } from "@knocklabs/react";
+
+export default function SlackChannelWrapper({
+  collection,
+  objectId,
+  className,
+}: {
+  collection: string;
+  objectId: string;
+  className: string;
+}) {
+  const slackChannelsRecipientObject = {
+    objectId: objectId,
+    collection: collection,
+  };
+  return (
+    <div className={className}>
+      <SlackChannelCombobox slackChannelsRecipientObject={slackChannelsRecipientObject} />
+    </div>
+  );
+}
+```
+
+As you select channels using the combobox input, the component will update the channel data for that object to store the Slack channel ids.
+
+![choose Slack channels](./images/choose-slack-channel.png)
+
+**Note: if you want to use a private Slack channel, you need to make sure that the Slack Bot has been invited to that channel.**
+
+At this point, Knock has all of the information it needs to send Slack messages to your selected channels. Click `Next` to examine the channel data for your object and tenant.
+
+## Examine channel data
+
+In this step, you're not required to take any action, but this page uses the Node SDK to pull channel data for your object recipient and tenant. Hopefully, this gives you an idea of how Knock is storing the data and these two abstractions interact.
```

Comment:
```suggestion
In this step, you're not required to take any action, but this page uses the Node SDK to pull channel data for your object recipient and tenant. This will give you an idea of how Knock is storing the data and these two abstractions interact.
```